docs/awsome.md

```diff
@@ -0,0 +1,154 @@
+
+## A Brief Anatomy of a Chart
+
+Inside of your newly created chart, there are three pieces:
+
+- The `Chart.yaml` file contains descriptive data about a chart.
+- The `manifests/` directory contains one or more Kubernetes manifest files.
+- A `README.md` file, which is a Markdown-formatted file that explains how to configure your chart.
+
+This guide walks you through the process of creating those parts.
+
+## Naming Your Chart
+
+A good name succinctly expresses what your chart provides. If your chart installs an application in the "normal" way or is a basic configuration of a package, you should name it with the name of the project. For example, a singe MongoDB chart may simple be called `mongo`.
+
+If your chart displays some more advanced features, or creates a cluster of servers, you should give it a more specific name: `redis-cluster` creates and configures a cluster or Redis servers.
+
+Please take a look at the existing charts before naming your own.
+
+_A Note on example charts:_ As Helm was getting started, we created a few example charts, whose purpose was to illustrate how to write a chart. We are trying to only add new _example_ charts when they illustrate something new and helpful for chart developers.
+
+## The Chart.yaml File
+
+A simple Chart.yaml file should look like this:
+
+```
+name: NAME OF CHART
+home: URL TO IMAGE HOME
+version: VERSION
+description: SHORT DESCRIPTION
+dependencies:
+  - name: CHART
+    version: SEMVER FILTER
+maintainers:
+  - YOUR NAME <YOUR EMAIL>
+details:
+  ONE PARAGRAPH DESCRIPTION
+```
+
+- name: This should be the same name as your chart directory.
+- home: The URL to the project where the container image came from. This is to assist people in finding out how their image was built.
+- version: A SemVer 2 version strings.
+- description: A short (several word) description of this chart.
+- dependencies: A set of name/filter pairs
+	- name: The name of the chart this chart depends upon
+	- version: A filter indicating what version of the chart is required. Example: `~1.2` (greater than or equal to 1.2.0, and less than 1.3.0)
+- maintainers: A set of maintainer names, together with an email
+- details: A single paragraph describing the chart
+
+Except for `dependencies`, all fields are required.
+
+## The README.md File
+
+The README file performs one important function: It tells the user how to use your chart. It is automatically displayed when a chart is installed.
+
+Your README.md file _may_:
+
+- Tell the user how to configure the chart (if necessary)
+- How to work with the chart once it's in Kubernetes (if this is not self-evident)
+- Explain any environment variables, secrets, annotations, or special labels
+- Provide URLs to external sources that are helpful
+
+Your REDME.md file _should not_:
+
+- Add a license constraint (all charts in `deis/charts` are Apache 2)
```

Comment:
What about charts in another repository i.e. `bacongobbler/charts`? Maybe we should change the wording around to imply that charts in deis/charts are Apache v2, but perhaps the chart.yml may specify this in its metadata?